kedro/framework/cli/starters.py

```diff
@@ -311,8 +312,18 @@ def starter() -> None:
 @click.option("--starter", "-s", "starter_alias", help=STARTER_ARG_HELP)
 @click.option("--checkout", help=CHECKOUT_ARG_HELP)
 @click.option("--directory", help=DIRECTORY_ARG_HELP)
-@click.option("--tools", "-t", "selected_tools", help=TOOLS_ARG_HELP)
-@click.option("--name", "-n", "project_name", help=NAME_ARG_HELP)
+@click.option(
+    "--name",
+    "-n",
+    "project_name",
+    help=NAME_ARG_HELP,
+)
+@click.option(
+    "--tools",
+    "-t",
+    "selected_tools",
+    help=TOOLS_ARG_HELP,
+)
 @click.option("--example", "-e", "example_pipeline", help=EXAMPLE_ARG_HELP)
```

Comment:
Can you use Click choice options here https://click.palletsprojects.com/en/stable/options/#choice-options to simplify the prompting and validation logic for example project?